### DISCUSSION TITLE
Can litelllm work with model serving with Triton inference server

### DISCUSSION THREAD

**Discussion by YunchaoYang** [BerriAI/litellm#3498] [Q&A]
Hi All,

just wondering that  is there a way for LittLM to infer with Triton Inference Server using the KServe protocol？

**Comment by ghost** [comment]
Hey @YunchaoYang i think i've seen this request before - do you have docs for the api? and is there an easy way to test it

**Comment by ishaan-jaff** [comment]
@YunchaoYang we support Triton Embeddings: https://docs.litellm.ai/docs/providers/triton-inference-server

Is this what you needed ?

**Reply by k0286** [reply]
Is there any plan to support completions api with TIS?

**Reply by ishaan-jaff** [reply]
hi @k0286 - Planning on adding support for it this week by Friday. Can we hop on a call to once this is live ? It would help to debug any issues you run into 

- my cal for your convenience: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat
- my linkedin if you prefer DMs: https://www.linkedin.com/in/reffajnaahsi/
